public/css/mixin/progress-bar.less

```
@@ -82,7 +78,8 @@
       }
 
       &.left {
-        margin-left: -6.75em;
+        margin-left: -2.75em;
+        width: 8em;
```

Again, neither of those is needed:

```suggestion
        // entire width (moves the right border in place of the left) + (.675em (:before placement) + .5em (half :before width)) + 1px (:before border)
        transform: translate(calc(-100% + 1.175em + 1px), 0);
```